php-base/composer.sh

```diff
@@ -77,10 +77,14 @@ EOF
     php -d auto_prepend_file='' /tmp/install_extensions.php ${APP_DIR}/composer.json ${PHP_DIR}/lib/conf.d/extensions.ini ${PHP_VERSION}
 
     # Run Composer.
+    if [ -n "${GOOGLE_RUNTIME_RUN_COMPOSER_SCRIPT}" ]; then
```

Comment:
Maybe we'll remove this when we remove the `php` image.

`--no-script` introduced for a user who was suffered from a fact that our build pipeline doesn't respect env_variables in app.yaml. 

So yes, when the builder is used, it always run the composer script.